examples/font.js

```diff
@@ -4,42 +4,32 @@
  */
 
 var Canvas = require('../lib/canvas')
-  , canvas = new Canvas(320, 320)
-  , Font = Canvas.Font
-  , ctx = canvas.getContext('2d')
   , fs = require('fs')
   , path = require("path");
 
-if (!Font) {
-	throw new Error('Need to compile with font support');
-}
-
 function fontFile(name) {
   return path.join(__dirname, '/pfennigFont/', name);
 }
 
-var pfennigFont = new Font('pfennigFont', fontFile('Pfennig.ttf'));
-pfennigFont.addFace(fontFile('PfennigBold.ttf'),   'bold');
-pfennigFont.addFace(fontFile('PfennigItalic.ttf'), 'normal', 'italic');
-pfennigFont.addFace(fontFile('PfennigBoldItalic.ttf'), 'bold', 'italic');
-
-var canvas = new Canvas(320, 320)
-var ctx = canvas.getContext('2d')
+Canvas.registerFont(fontFile('Pfennig.ttf'));
```

Comment:
Suggest adding a comment here to explain that the file name itself is irrelevant and that the font is found based on the name of the font in the ttf file itself.